armbian/base/customize-armbian-rockpro64.sh

```diff
@@ -302,30 +302,30 @@ if [[ "${BASE_BUILDMODE}" != "ondevice" ]] && [[ "${BASE_MINIMAL}" == "true" ]];
 
   for pkg in $pkgToRemove
   do
-    apt -y remove "$pkg" || true
+    apt-get -y purge "$pkg" || true
```

Comment:
What is the reason that all of them cannot be purged in one go?

Reply:
See original issue
https://github.com/shiftdevices/bitbox-base-internal/issues/404#issuecomment-579718816